src/components/organisms/footer/index.tsx

```diff
@@ -0,0 +1,9 @@
+import tw, { css } from "twin.macro";
+import { Text } from "../../atoms/text";
+const Footer: React.FC = () => (
+  <footer css={tw`absolute bottom-0 w-screen`}>
```

Comment:
後々死ぬのでabsoluteはやめましょう

```suggestion
  <footer css={tw`bottom-0 w-screen`}>
```